commcare_connect/program/utils.py

```diff
@@ -21,3 +21,50 @@ def is_program_manager_of_opportunity(request, opp_id) -> bool:
         and managed_opp.program.organization.slug == request.org.slug
         and is_program_manager(request)
     )
+
+
+def populate_currency_and_country_fk_for_model(apps, model_name, app_label, total_label):
+    """
+    Migration util to populate currency_fk and country fields for a opportunity/program
```

Comment:
nit: If this is just for use by the migration, I think it can go inline in the migration file

Reply:
There are two models `Opportunity` and `Program` that have currency field. So this is used in those two migration files as a common helper.